src/components/AccountSwitcher.tsx

```diff
@@ -47,31 +47,30 @@ function AccountSwitcher() {
     const isActingAsDelegate = !!account?.delegatedAccess?.delegate ?? false;
     const canSwitchAccounts = canUseNewDotCopilot && (delegators.length > 0 || isActingAsDelegate);
 
-    const createBaseMenuItem = (personalDetails: PersonalDetails | undefined, errors?: Errors, additionalProps: MenuItemWithLink = {}): MenuItemWithLink => {
+    const createBaseMenuItem = (personalDetails: PersonalDetails | undefined, errors?: Errors, additionalProps: Partial<PopoverMenuItem> = {}): PopoverMenuItem => {
         const error = Object.values(errors ?? {})[0] ?? '';
         return {
-            title: personalDetails?.displayName ?? personalDetails?.login,
+            text: personalDetails?.displayName ?? personalDetails?.login ?? '',
             description: Str.removeSMSDomain(personalDetails?.login ?? ''),
             avatarID: personalDetails?.accountID ?? -1,
             icon: personalDetails?.avatar ?? '',
-            iconType: CONST.ICON_TYPE_AVATAR,
             outerWrapperStyle: shouldUseNarrowLayout ? {} : styles.accountSwitcherPopover,
             numberOfLinesDescription: 1,
             errorText: error ?? '',
             shouldShowRedDotIndicator: !!error,
             errorTextStyle: styles.mt2,
             ...additionalProps,
+            iconType: CONST.ICON_TYPE_AVATAR,
```

Comment:
Any reason for moving this?

Reply:
We need to moving this here to prevent the type error.

Reply:
The error is because we can send `iconType` via `additionalProps` and it would conflict with the set `icon`. Moving the prop to be after the `additionalProps` is not the correct approach because this will still mislead you that you can pass `iconType` into `additionalProps` but in fact what you pass will be overwritten.

A better approach is to just omit the props not to mislead other developers 
```ts
additionalProps?: Partial<Omit<PopoverMenuItem, 'icon' | 'iconType'>>
```